Makefile

```
@@ -14,11 +14,13 @@ build: generate
 test: FORCE
 	go test ./... -ldflags="-X 'github.com/superfly/flyctl/internal/buildinfo.buildDate=$(NOW_RFC3339)'" --run=$(T)
 
-# to run one test, use: make preflight-test T=TestAppsV2ConfigSave
-preflight-test: build
+raw-preflight-test:
 	if [ -r .direnv/preflight ]; then . .direnv/preflight; fi; \
 	go test ./test/preflight --tags=integration -v -timeout 10m --run="$(T)"
 
+# to run one test, use: make preflight-test T=TestAppsV2ConfigSave
+preflight-test: build raw-preflight-test
```

```suggestion
preflight-test: build
	$(MAKE) raw-preflight-test
```

it's possible that make can run dependency targets in parallel, so this change ensures the build happens before the tests. probably won't matter in most cases...